Add note in `CONTRIBUTING.md` about not asking to be assigned to issues (#8871)
TheAlgorithms/Python: CONTRIBUTING.md

Your contribution will be tested by our [automated testing on GitHub Actions](https://github.com/TheAlgorithms/Python/actions) to save time and mental energy.  After you have submitted your pull request, you should see the GitHub Actions tests start to run at the bottom of your submission page.  If those tests fail, then click on the ___details___ button try to read through the GitHub Actions output to understand the failure.  If you do not understand, please leave a comment on your submission page and a community member will try to help.

> If you are interested in resolving an [open issue](https://github.com/TheAlgorithms/Python/issues), simply make a pull request with the fix. __We do not assign issues in this repo__ so please do not ask for permission to work on the issue.

Reviewer: ```suggestion
If you are interested in resolving an [open issue](https://github.com/TheAlgorithms/Python/issues), simply make a pull request with your proposed fix. __We do not assign issues in this repo__ so please do not ask for permission to work on an issue.
```

> If you are interested in resolving an [open issue](https://github.com/TheAlgorithms/Python/issues), simply make a pull request with the fix. __You do not need to ask for permission to work on the issue, as we do not assign issues in this repo.__

Reviewer: ```suggestion
If you are interested in resolving an [open issue](https://github.com/TheAlgorithms/Python/issues), simply make a pull request with the fix. __We do not assign issues in this repo__ so please do not ask for permission to work on the issue.
```
